src/utils/uuid.ts

```diff
@@ -0,0 +1,7 @@
+export function uuidv4(): string {
```

Comment:
I'd recommend just installing the `uuid` package on npm, this script has been [complained about on Twitter](https://twitter.com/stijnveilig/status/1409120079648808962?s=20) by infosec folks for reasons I don't fully understand

Reply:
That's an interesting take. I've asked the author to elaborate on the precise reasons the snippet is unrecommended. I don't mind installing an extra dependency, but I like understanding things first.